### DISCUSSION TITLE
Allow reordering of categories

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#2935] [Discussions] [answered]
It would be really nice, if we could re-arange the order in which the categories are displayed on the side, without having to rename (or delete and re-add) categories.

**Comment by becca** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/2867

**Reply by tisonkun** [reply]
Not quite a duplicate. #2867 wants a different sort strategy, but this issue wants any order manually.

I suppose this is natural, like you can drag and change the input method order for a mobile phone >_<

**Reply by tisonkun** [reply]
I have a "Q&A" category, and I have a "General" category. For user friendly purpose, I'd like to put "Q&A" first and "General" below.

Currently, no way to do it.
